smack-extensions/src/main/java/org/jivesoftware/smackx/jingle/JingleManager.java

```diff
@@ -33,6 +28,11 @@
 import org.jxmpp.jid.FullJid;
 import org.jxmpp.jid.Jid;
 
+import java.util.Map;
```

Comment:
Non semantic re-order change of the imports. Please revert that. And avoid that in the future. The (eclipse) default is roughly
1.static
2. java.*
3. org.*

See also https://stackoverflow.com/questions/14716283/is-it-possible-for-intellij-to-organize-imports-the-same-way-as-in-eclipse